### DISCUSSION TITLE
Cross origin redirect is not supported, so what we need to do ?

### DISCUSSION THREAD

**Discussion by ekimcem** [supabase/supabase#30918] [Questions]
In new versions of many browsers now we need to show a pop up for social logins, for example in firebase it warns:

caution:
Cross origin redirect sign-in is no longer supported in many browsers. Update your app to ensure your users can continue to sign into your app.

I am using twitter authenticaion with supabase, now it is not working with new versions of chrome.

What should we do for this ?

**Comment by GaryAustin1** [comment]
You might want to file this in supabase/auth-js issues.
